.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
-	branch = bleeding
+	url = https://github.com/sanman00/SpongeAPI.git
+	branch = sleeping-data-impl
```

Comment:
This comes from the first commit. Many pull requests seem to modify this file to point to a fork when they modify the API and implementation so that they can compile.